docs/source/create_channel/create_channel.md

```diff
@@ -0,0 +1,383 @@
+
+You can visit the [Using configtx.yaml to create a channel genesis block](create_channel_genesis.html) tutorial to learn more about this file. For now, we will return to the operational aspects of creating the channel, though we will reference parts of this file in future steps.
+
+## Start the network
+
+We will use a running instance of the Fabric test network to create the new channel. For the sake of this tutorial, we want to operate from a known initial state. The following command will kill any active containers and remove any previously generated artifacts. Make sure that you are still operating from the `test-network` directory of your local clone of `fabric-samples`.
+```
+./network.sh down
+```
+You can then use the following command to start the test network:
+```
+./network.sh up
+```
+This command will create a Fabric network with the two peer organizations and the single ordering organization defined in the `configtx.yaml` file. The peer organizations will operate one peer each, while the ordering service administrator will operate a single ordering node. If the command completes successfully, the script will print out logs of the nodes being created:
+```
+Creating network "net_test" with the default driver
+Creating volume "net_orderer.example.com" with default driver
+Creating volume "net_peer0.org1.example.com" with default driver
+Creating volume "net_peer0.org2.example.com" with default driver
+Creating orderer.example.com    ... done
+Creating peer0.org2.example.com ... done
+Creating peer0.org1.example.com ... done
+CONTAINER ID        IMAGE                               COMMAND             CREATED             STATUS                  PORTS                              NAMES
+8d0c74b9d6af        hyperledger/fabric-orderer:latest   "orderer"           4 seconds ago       Up Less than a second   0.0.0.0:7050->7050/tcp             orderer.example.com
+ea1cf82b5b99        hyperledger/fabric-peer:latest      "peer node start"   4 seconds ago       Up Less than a second   0.0.0.0:7051->7051/tcp             peer0.org1.example.com
+cd8d9b23cb56        hyperledger/fabric-peer:latest      "peer node start"   4 seconds ago       Up 1 second             7051/tcp, 0.0.0.0:9051->9051/tcp   peer0.org2.example.com
+```
+
+The `./network.sh up` deploys an instance of the test network without creating an application channel. However, the orderer system channel is created by the test network script in order to deploy the network, and does not need to be created as step in this tutorial. However, because the system channel is used to create other channels, it is important to understand the orderer system channel before we create an application channel.
```

Comment:
Change this paragraph to: Our instance of the test network was deployed without creating an application channel. However, the test network script creates the system channel when you issue the `./network.sh up` command. Under the covers, the script uses the `configtxgen` tool and the `configtx.yaml` file to build the genesis block of the system channel.  Because the system channel is used to create other channels, we need to take some time to understand the orderer system channel before we can create an application channel.